### PR TITLE
feat(vinnie): bump pizza up and add veal

### PR DESCRIPTION
**Pull request by Porthorian**

### CODE REVIEW

File: vinnie.md [hunk 1]
```diff
@@ -6,18 +6,19 @@
...
+12. Pasta
+13. Mac+Cheese
+14. Calamari
+15. Sausage + Peppers
+16. Pulled Pork
+17. Fried Rice
+18. BLT
+19. Veal
+20. Sushi
+21. Shrimp
```
Comment (ddymko) [inline vinnie.md:21]: HOW IS THIS 21

File: vinnie.md [hunk 2]
```diff
@@ -6,18 +6,19 @@
 6. Porkroll, egg, and cheese on plain/sandwich
 7. Buffalo Chicekn Wrap but counts as #2
```
Comment (zamerendino) [inline vinnie.md:7]: this implies buffalo chicken is your least favorite kind of chicken though?

File: vinnie.md [hunk 3]
```diff
@@ -6,18 +6,19 @@
...
+14. Calamari
+15. Sausage + Peppers
+16. Pulled Pork
+17. Fried Rice
+18. BLT
+19. Veal
+20. Sushi
+21. Shrimp
+22. Fish 
+23. Stromboli
```
Comment (dean-haw) [inline vinnie.md:23]: Stromboli is too similar to pizza and should be trailing behind pizza a bit more. Now that Pizza is #9 this should be trailing at least 5 spots behind!

Comment (Porthorian) [reply vinnie.md:23]: Your wrong, stromboli is good but also not a top 10 that I would wanna get usually.

File: vinnie.md [hunk 4]
```diff
@@ -6,18 +6,19 @@
...
-16. Fried Rice
-17. BLT
-18. Pizza
-19. Sushi
-20. Shrimp
-21. Fish 
-22. Stromboli
+9. Pizza
+10. Ribs
+11. Surf & Turf
```
[the remark this answers is not in the document]
Comment (dean-haw) [reply vinnie.md:11]: Vinnie needs to make up his mind if Lobster and Steak is really # 1. I think he's proved that steak is his favorite. No doubts on that. But make up your mind about Surf and Turf / lobster

[the remark this answers is not in the document]
Comment (Porthorian) [reply vinnie.md:11]: Your wrong zach

File: vinnie.md (hunk 1 above)
Comment (Porthorian) [reply vinnie.md:21]: Tough

File: vinnie.md (hunk 4 above)
[the remark this answers is not in the document]
Comment (zamerendino) [reply vinnie.md:11]: you're*

File: vinnie.md (hunk 2 above)
Comment (zamerendino) [reply vinnie.md:7]: but putting it at #7 implies that any other type of chicken (even alive) out ranks it at #2?

File: vinnie.md (hunk 3 above)
Comment (ddymko) [reply vinnie.md:23]: > not a top 10 that I would wanna get usually
dude clams don't belong on that list then

File: vinnie.md (hunk 4 above)
[the remark this answers is not in the document]
Comment (Porthorian) [reply vinnie.md:11]: @zamerendino grammar nazi over here

File: vinnie.md (hunk 3 above)
Comment (aanthonyrizzo) [reply vinnie.md:23]: Im going to have to agree with vinnie on this one. Stromboli is different enough from pizza that I agree its ranking should be decoupled

**Pull request merged**